<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transition Color</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
   <item>
    <widget class="ColorWidget" name="mainColorWidget" native="true"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,1,0,2">
     <item>
      <widget class="QLabel" name="redLabel">
       <property name="text">
        <string>Red:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="redSlider">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="greenLabel">
       <property name="text">
        <string>Green:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="greenSlider">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="blueLabel">
       <property name="text">
        <string>Blue:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="blueSlider">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ColorWidget" name="previewColorWidget" native="true"/>
     </item>
     <item>
      <widget class="QPushButton" name="goButton">
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorWidget</class>
   <extends>QWidget</extends>
   <header location="global">colorwidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>redSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>targetColorChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>targetColorChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>targetColorChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>goButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>goButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>targetColorChanged()</slot>
  <slot>goButtonClicked()</slot>
 </slots>
</ui>
